app/reports/page-content.tsx

```diff
@@ -126,20 +128,29 @@ const ResolvedFilters = () => {
         {
           id: 'takendown',
           text: 'Taken Down',
-          onClick: () => updateParams('takendown', true),
+          onClick: () => updateParams({ takendown: true }),
           isActive: takendown === 'true',
         },
         {
-          id: 'includeMuted',
-          text: 'Show Muted',
-          onClick: () => updateParams('includeMuted', true),
-          isActive: includeMuted === 'true',
-        },
-        {
-          id: 'onlyMuted',
-          text: 'Only Muted',
-          onClick: () => updateParams('onlyMuted', true),
-          isActive: onlyMuted === 'true',
+          id: 'mute',
+          text:
+            includeMuted === 'true'
+              ? 'Include Muted'
+              : onlyMuted === 'true'
+              ? 'Only Muted'
+              : 'Mutes',
+          onClick: () => {
+            // setting a param to it's current value toggles it off
+            // so we toggle off includeMuted and toggle on onlyMuted
+            if (includeMuted === 'true') {
+              updateParams({ includeMuted: true, onlyMuted: true })
```

Comment:
We'd move mods of these into the new useQueueFilters hook but trying not to bite into too much in one go.